### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by mark-prins** [openmsupply/msupply-dashboard#66]
Some possible options:

1. Modify the notifications that run from the mSupply dashboard to be able to iterate
2. make a new module that can do this: perhaps a node app with config interface that runs on the server. takes data from postgres
3. Something out of grafana. plugin possibly?

**Comment by mark-prins** [comment]
g3adfc54

**Comment by CarlosNZ** [comment]
Is this for notifying about stock levels, that kind of thing? Can we not use Grafana's built-in Alerts?

**Comment by mark-prins** [comment]
yes, and no. The alerts in grafana aren't fine grained enough and won't allow for tweaking per user.. I think that was the issue. There's a reason that they aren't good enough anyway. I've copied in text from an email exchange on the topic and really don't know what I'm talking about above
